extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts

```diff
@@ -80,8 +80,9 @@ export class ChePluginServiceImpl implements ChePluginService {
 
         try {
             const workspaceSettings: WorkspaceSettings = await this.cheApiService.getWorkspaceSettings();
-            if (workspaceSettings && workspaceSettings['cheWorkspacePluginRegistryUrl']) {
-                let uri = workspaceSettings['cheWorkspacePluginRegistryUrl'];
+            if (workspaceSettings && workspaceSettings['cheWorkspacePluginRegistryInternalUrl']) {
+                let uri = workspaceSettings['cheWorkspacePluginRegistryInternalUrl'];
```

Comment:
should be fallback to `cheWorkspacePluginRegistryUrl` if `cheWorkspacePluginRegistryInternalUrl` isn't set?

Reply:
Yes, we have fallback on the che-server side. And If user set up external registry, we don't override it to internal link.

Reply:
If it's handled on the bakend, we don't need the variable name. WDYT?